CRISPResso2/CRISPRessoMultiProcessing.py

```diff
@@ -40,7 +40,13 @@ def run_crispresso(crispresso_cmds, descriptor, idx):
         logging.info('Finished CRISPResso %s #%d' %(descriptor, idx))
     return return_value
 
-def run_crispresso_cmds(crispresso_cmds,n_processes="1",descriptor = 'region',continue_on_fail=False):
+
+def wrapper(func, args):
+    idx, args = args
+    return (idx, func(args))
+
+
+def run_crispresso_cmds(crispresso_cmds, logger, n_processes="1", descriptor = 'region', continue_on_fail=False, start_end_percent=None):
```

Comment:
double check that all other instances of run_crispresso_cmds are updated as well

consider moving logger to optional with default None

Reply:
also, can I request a commenting?

Reply:
I discovered the incantation to see which module called a function and use that to get the correct logger object now :)